\documentclass{article}
\usepackage{times}
\usepackage[english]{babel}
\usepackage[utf8]{inputenc}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{setspace}
\usepackage{verbatim}
\usepackage{graphicx}
%\usepackage{mathpazo}
\usepackage[hmargin=8em,vmargin=8em]{geometry}
\usepackage{enumerate}
\usepackage{setspace}
\usepackage{listings}
\usepackage{mathrsfs}

%New definition of square root:
% it renames \sqrt as \oldsqrt
\let\oldsqrt\sqrt
% it defines the new \sqrt in terms of the old one
\def\sqrt{\mathpalette\DHLhksqrt}
\def\DHLhksqrt#1#2{%
\setbox0=\hbox{$#1\oldsqrt{#2\,}$}\dimen0=\ht0
\advance\dimen0-0.2\ht0
\setbox2=\hbox{\vrule height\ht0 depth -\dimen0}%
{\box0\lower0.4pt\box2}}

\newcommand{\listfile}[1]{\lstset{caption=#1,basicstyle=\small, breaklines=true, breakatwhitespace=false}
  \lstinputlisting{#1}}
\lstset{language=R, frame=single, breaklines=true}

\newcommand{\lp}{\left (}
\newcommand{\rp}{\right )}

\begin{document}

\title {\onehalfspacing Master Project Proposal \\ \singlespacing Implementing and Evaluating a Statistical Model for Clustering Metagenomic Contigs}
\author{Brynjar Sm√°ri Bjarnason 840824-4690\\Computational and Systems Biology\\ School of Computer Science and Communication\\KTH Royal Institute of Technology \\ \\ 
        Supervisor: Anders Andersson\\School of Biotechnology \\ KTH Royal Institute of Technology\\ \\
        Co-supervisor: Christopher Quince\\School of Engineering\\University of Glasgow}

\maketitle

\section*{The Problem}
In metagenomic research, which is an analysis of microbial communities from environmental samples, we want to know which organisms are found in samples such as water, soil and human gut. Most of these organisms have not been sequenced and many can not be cultured in laboratory. With massively parallel sequencing (MPS) we get short reads of all genetic material in these samples which are assembled into continuous genetic strands called contigs. Each contig represents a fragment of an organisms genome, and to reconstruct the genomes in the sample the contigs need to be clustered into bins. This is referred to as the binning process. After this process, each bin should contain only contigs from one organism. This allows us to examine which organisms are in the sample, in what abundance and mutations within organism.

The goal with this project (and a project run in parallel) is to develop a rigorous statistical model and efficient implementation for a clustering method using multiple samples from the same environment taken over period of time or from close regions. Using multiple samples will hopefully give us better view of the organisms in that environment since microbial communities vary over time and place and contigs from the same organism should co-vary in abundance across samples (as opposed to contigs from different organisms). 

\section*{Description}
The focus of this project is implementation and optimization of a statistical model for clustering contigs from multiple samples. The statistical model will be designed in parallel so many models need to be compared for quality and efficiency, also comparison to current binning projects should be done. Therefore a framework for comparing different models and implementations will be developed. 

\end{document}
